src/app/src/components/FacilityDetailSidebarDetail.jsx

```diff
@@ -13,56 +11,27 @@ const detailsSidebarStyles = () =>
         primaryText: {
             wordWrap: 'break-word',
         },
-        secondaryText: {
-            color: 'rgba(0, 0, 0, 0.54)',
-            display: 'flex',
-            alignItems: 'center',
-            fontSize: '12px',
-            justify: 'flex-end',
-        },
-        divider: {
-            backgroundColor: 'rgba(0, 0, 0, 0.06)',
+        item: {
+            boxShadow: '0px -1px 0px 0px rgb(240, 240, 240)',
         },
     });
 
 const FacilityDetailSidebarDetail = ({
-    hasAdditionalContent,
-    additionalContentCount,
     primary,
     secondary,
-    onClick,
-    hideTopDivider,
     verified,
     classes,
 }) => (
-    <>
-        <ShowOnly when={!hideTopDivider}>
-            <Divider className={classes.divider} />
+    <ListItem className={classes.item}>
+        <ShowOnly when={verified}>
```

Comment:
I'm showing the verified badge when an extended field is verified (presumably includes when the contributor is verified, based on upcoming changes). Do we want this, or should there be no badge for verified items?

Reply:
It makes sense that we visually indicate the status of these items. I like shoing the badge.